packages/indexer-nibi/src/heart-monitor.ts

```diff
@@ -128,8 +128,8 @@ export class HeartMonitor implements IHeartMonitor {
       this.gqlEndpt = this.defaultGqlEndpt
     } else if (typeof gqlEndpt === "string") {
       this.gqlEndpt = gqlEndpt
-    } else if (chain?.endptTm !== undefined) {
-      const endptFromRpc: string | null = gqlEndptFromTmRpc(chain?.endptTm)
+    } else if (chain && chain.endptTm !== undefined) {
```

Comment:
Optional chaining should be fine here

`if (chain?.endptTm)`

Reply:
Yeah, but I noticed that Optional chaining decreased our branch coverage for the file, and this change increases it.

Reply:
Probably because it makes a new logic branch :(

Reply:
Fine for now